controllers/searchController.js

```diff
@@ -0,0 +1,122 @@
+import { Op } from 'sequelize';
+import ServerResponse from '../modules';
+import models from '../database/models';
+
+const { User, Article, Tag } = models;
+const { successResponse, errorResponse } = ServerResponse;
+
+/**
+ *
+ *
+ * @export
+ * @class ArticleController
+ */
+class SearchController {
+/**
+ * perform custom search sitewide
+ *
+ * @param {Object} req Express request
+ * @param {Object} res Express object
+ * @param {Object} next Express object
+ * @returns {Array} articles, authors and tags
+ */
+  static async customSearch(req, res, next) {
+    const { keyword } = req.query;
+
+    if (!keyword) {
+      return errorResponse(res, 400, 'Please input a search parameter');
```

Comment:
Are you passing `Please input a search parameter` as the key here? I think this would be difficult to access in the frontend, you can just use `search` then pass the sentence as a message